packages/caver-core-helpers/src/validateFunction.d.ts

```diff
@@ -0,0 +1,25 @@
+/*
+    Copyright 2021 The caver-js Authors
+    This file is part of the caver-js library.
+
+    The caver-js library is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    The caver-js library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with the caver-js. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { TransactionObject } from '../../caver-rtm/src'
+
+export default interface IValidateFunction {
+    validateParams(tx: TransactionObject): Error
```

Comment:
```suggestion
    validateParams(tx: TransactionObject | Transaction): Error
```

I understand that TransactionObject defined in `caver-trm` is an object containing the result of `caver.rpc.getTransaction`. So I think the TransactionObject used here should come from `caver-klay-accounts`.

Reply:
Is that `Transaction` meaning **type** `Transaction` at caver-transaction/src/index.d.ts?
I'm confusing with **interface** `Transaction` at packages/caver-rtm/src/index.d.ts  that you suggested above 😅

Reply:
I mean Transaction at `caver-transaction/src/index.d.ts`.

Reply:
Fixed it. please check 100f67d